content/id/docs/concepts/services-networking/service.md

```diff
@@ -0,0 +1,1065 @@
+Sertifikat yang digunakan bisa saja berasal dari _third party_ yang diunggah ke IAM atau 
+sertifikat yang dibuat secara langsung dengan menggunakan sertifikat manajer AWS.
+
+```yaml
+metadata:
+  name: my-service
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-backend-protocol: (https|http|ssl|tcp)
+```
+
+Anotasi kedua memberikan spesifikasi bagi protokol yang digunakan oleh _Pod_ untuk saling berkomunikasi.
+Untuk HTTPS dan SSL, ELB membutuhkan _Pod_ untuk melakukan autentikasi terhadap dirinya sendiri melalui 
+koneksi yang dienkripsi.
+
+Protokol HTTP dan HTTPS akan memilih mekanisme _proxy_ di tingkatan ke-7: 
+ELB akan melakukan terminasi koneksi dengan pengguna, melakukan proses _parsing_ _headers_, serta 
+memasukkan _value_ bagi _header_ _X-Forwarded-For_ dengan alamat IP pengguna (_Pod_ hanya dapat melihat 
+alamat IP dari ELB pada akhir koneksi yang diberikan) ketika melakukan _forwarding_ suatu _request_.
+ 
+Protokol TCP dan SSL akan memilih mekanisme _proxy_ pada tingkatan 4: ELB akan melakukan _forwarding_ trafik 
+tanpa melakukan modifikasi _headers_.
+
+Pada _environment_ campuran dimana beberapa _port_ diamankan sementara _port_ lainnya dalam kondisi tidak dienkripsi, 
+anotasi-anotasi berikut dapat digunakan:
+
+```yaml
+    metadata:
+      name: my-service
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
+        service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443,8443"
+```
+
+Pada contoh di atas, jika _Service_ memiliki 3 buah _port_, yaitu: _80_, _443_, dan 
+_8443_, maka _443_ adan _8443_ akan menggunakan sertifikat SSL, tetapi _80_ hanya akan 
+di-_porxy_ menggunakan protokol HTTP.
+
+Mulai versi 1.9, _Service_ juga dapat menggunakan [_predefined_ _policy_](http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-security-policy-table.html)
+untuk HTTPS atau _listener_ SSL. Untuk melihat _policy_ apa saja yang dapat digunakan, kamu dapat menjalankan perintah _awscli_:
+
+```bash
+aws elb describe-load-balancer-policies --query 'PolicyDescriptions[].PolicyName'
+```
+
+_Policy_ ini kemudian dapat dispesifikasikan menggunakan anotasi 
+"_service.beta.kubernetes.io/aws-load-balancer-ssl-negotiation-policy_", contohnya:
+
+```yaml
+    metadata:
+      name: my-service
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-ssl-negotiation-policy: "ELBSecurityPolicy-TLS-1-2-2017-01"
+```
+
+#### Protokol PROXY pada AWS
+
+Untuk mengaktifkan dukungan [protokol PROXY](https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt)
+untuk kluster yang dijalankan di AWS, kamu dapat menggunakan anotasi di bawah ini:
+
+```yaml
+    metadata:
+      name: my-service
+      annotations:
+        service.beta.kubernetes.io/aws-load-balancer-proxy-protocol: "*"
+```
+
+Sejak versi 1.3.0, penggunaan anotasi berlaku untuk semua _port_ yang diproksi oleh ELB 
+dan tidak dapat diatur sebaliknya.
+
+#### Akses _Log_ ELB pada AWS
```

Comment:
annotation values should all be in backtick, follow https://deploy-preview-14117--kubernetes-io-master-staging.netlify.com/docs/concepts/services-networking/service/#elb-access-logs-on-aws